data/get_drive_file.py

```diff
@@ -0,0 +1,42 @@
+import requests
+
+def download_file_from_google_drive(id, destination):
+    def get_confirm_token(response):
+        for key, value in response.cookies.items():
+            if key.startswith('download_warning'):
+                return value
+
+        return None
+
+    def save_response_content(response, destination):
+        CHUNK_SIZE = 32768
```

Comment:
Try stick to the PEP8 naming conventions. It keeps things clean and regular across the code base

Reply:
ie. ```CHUNK_SIZE``` should be in snake case